pkg/server/controllers.go

```diff
@@ -680,6 +632,70 @@ func (s *Server) installAPIBindingController(ctx context.Context, config *rest.C
 	})
 }
 
+func (s *Server) installAPIBinderController(ctx context.Context, config *rest.Config, server *genericapiserver.GenericAPIServer) error {
+	// Client used to create APIBindings within the initializing workspace
+	config = rest.CopyConfig(config)
+	kcpclienthelper.SetMultiClusterRoundTripper(config)
+	config = rest.AddUserAgent(config, initialization.ControllerName)
+	config.Host = fmt.Sprintf("https://%v/services/%v/%v", s.GenericConfig.ExternalAddress, initializingworkspaces.VirtualWorkspaceName, tenancyv1alpha1.ClusterWorkspaceAPIBindingsInitializer)
+	initializingWorkspacesKcpClusterClient, err := kcpclient.NewForConfig(config)
+	if err != nil {
+		return err
+	}
+
+	// Wildcard client used for informers
+	informerCfg := rest.CopyConfig(config)
+	kcpclienthelper.SetCluster(informerCfg, logicalcluster.Wildcard)
+	informerClient, err := kcpclient.NewForConfig(informerCfg)
+	if err != nil {
+		return err
+	}
+	initializingWorkspacesKcpInformers := kcpexternalversions.NewSharedInformerFactoryWithOptions(
+		informerClient,
+		resyncPeriod,
+		kcpexternalversions.WithExtraClusterScopedIndexers(indexers.ClusterScoped()),
+		kcpexternalversions.WithExtraNamespaceScopedIndexers(indexers.NamespaceScoped()),
+	)
```

Comment:
why this additional informer? And why here?

Reply:
do you want to inform on all workspaces and only act on those that are initializing? otherwise, we need this informer on the init ws vw. we can move its instantiation up if you want

Reply:
chatted on meet; this is ok